### PR TITLE
[docs] Translate the prometheus monitoring module

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

Translation of the `CustomAlertmanager` section.

## Changelog entries

```changes
section: docs
type: fix
summary: "Translation of the `CustomAlertmanager` section"
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/010-prometheus-crd/crds/customalertmanager.yaml [hunk 1]
```diff
@@ -99,34 +100,30 @@ spec:
...
-                            description: Labels that must have an equal value in the source
-                              and target alert for the inhibition to take effect.
+                            description: Labels that must have an equal value in the source and target alert for the inhibition to take effect.
                             items:
                               type: string
                             type: array
                           sourceMatch:
-                            description: Matchers for which one or more alerts have to exist
-                              for the inhibition to take effect. The operator enforces that
-                              the alert matches the resource’s namespace.
```
Comment (RomanenkoDenys) [inline modules/010-prometheus-crd/crds/customalertmanager.yaml:119]: Why we remove "The operator enforces that the alert matches the resource’s namespace." ?

File: modules/010-prometheus-crd/crds/customalertmanager.yaml [hunk 2]
```diff
@@ -136,31 +133,19 @@ spec:
                               type: object
                             type: array
                           targetMatch:
-                            description: Matchers that have to be fulfilled in the alerts
-                              to be muted. The operator enforces that the alert matches
-                              the resource’s namespace.
```
Comment (RomanenkoDenys) [inline modules/010-prometheus-crd/crds/customalertmanager.yaml:141]: Same question

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml [hunk 3]
```diff
@@ -37,1236 +45,366 @@ spec:
...
+                  description: Параметры настройки внутренного Alertmanager.
                   properties:
                     inhibitRules:
-                      description: List of inhibition rules. The rules will only apply to
-                        alerts matching the resource’s namespace.
+                      description: Список правил запрета.
                       items:
-                        description: InhibitRule defines an inhibition rule that allows
-                          to mute alerts when other alerts are already firing. See https://prometheus.io/docs/alerting/latest/configuration/#inhibit_rule
+                        description: Правила запрета используются для подавления уведомлений от определенных оповещений, если уже сработали другие определенные оповещения ([подробнее...](https://prometheus.io/docs/alerting/latest/configuration/#inhibit_rule)).
```
Comment (RomanenkoDenys) [inline modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml:56]: I think we do not translate inhibitRule word.

File: modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml [hunk 4]
```diff
@@ -37,1236 +45,366 @@ spec:
...
                         properties:
                           equal:
-                            description: Labels that must have an equal value in the source
-                              and target alert for the inhibition to take effect.
-                            items:
-                              type: string
-                            type: array
+                            description: Лейблы, которые должны иметь одинаковое значение в исходном и целевом алерте, чтобы запрет вступил в силу.
```
Comment (RomanenkoDenys) [inline modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml:59]: use word inhibit

**Review by RomanenkoDenys — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/010-prometheus-crd/crds/customalertmanager.yaml (hunk 1 above)
Comment (z9r5) [reply modules/010-prometheus-crd/crds/customalertmanager.yaml:119]: Fixed

File: modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml [hunk 5]
```diff
@@ -37,1236 +45,366 @@ spec:
...
-                              type: object
-                            type: array
-                        type: object
-                      type: array
+                              description: Условие содержит критерии удовлетворяющих ему лейблов алертов.
+                              properties: *match-properties
                     receivers:
-                      description: List of receivers.
+                      description: |
+                        Список получателей.
```
[the remark this answers is not in the document]
Comment (z9r5) [reply modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml]: Fixed

File: modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml [hunk 6]
```diff
@@ -37,1236 +45,366 @@ spec:
...
+                                        Ключ (имя ключа) Secret'а, из которого брать данные.
                                     name:
-                                      description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
-                                        TODO: Add other useful fields. apiVersion, kind,
-                                        uid?'
-                                      type: string
+                                      description: |
+                                        Ссылка на Secret.
+
+                                        [Подробнее о формате...](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names)
```
[the remark this answers is not in the document]
Comment (z9r5) [reply modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml]: Fixed

File: modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml (hunk 3 above)
Comment (z9r5) [reply modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml]: Fixed

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by z9r5** [comment]
Cherry-picked into release-1.34.

**Pull request merged**